po/cs.po

```diff
@@ -0,0 +1,141 @@
+# Czech translation for flatseal package.
+# Copyright (C) 2020 THE flatseal'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the flatseal package.
+#
+# Daniel Rusek <mail@asciiwolf.com>, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: flatseal\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-02-02 22:58-0300\n"
+"PO-Revision-Date: 2020-02-03 14:58+0100\n"
+"Language: cs\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: Daniel Rusek <mail@asciiwolf.com>\n"
+"Language-Team: \n"
+"X-Generator: Poedit 2.2.4\n"
```

Comment:
Last comment, try using `ninja flatseal-update-po` to generate the file, so it will already include the plurals entry.

Reply:
Done. I will update the PR to reflect these changes.